### PR TITLE
modified playlist object and created playlist factory

### PR DESCRIPTION
**Pull request by NicholasMacasaet**
fixed playlist class and added a playlist factory class

### CODE REVIEW

**Review by rafaelgacesa — APPROVED** [review]
Thanks Nick! Love this code. 👨‍💻

**Review by Mistryr222 — APPROVED** [review]
Excellent changes, I literally did this with you :D

File: src/main/java/Entities/PlaylistFactory.java [hunk 1]
```diff
@@ -1,4 +1,7 @@
 package Entities;
 
 public class PlaylistFactory {
+    public Playlist createPlaylist(String name, User owner, boolean isPublic){
+        return new Playlist(name,owner,isPublic);
```
Comment (sarxhlwalker) [inline src/main/java/Entities/PlaylistFactory.java:5]: Please put spaces after the commas in the constructor call

**Review by sarxhlwalker — CHANGES_REQUESTED** [review]
Please see comments.

**Review by sarxhlwalker — APPROVED** [review]
Gorgeous code!

File: src/main/java/Entities/PlaylistFactory.java (hunk 1 above)
Comment (rafaelgacesa) [reply src/main/java/Entities/PlaylistFactory.java]: In Java, white space does not matter. In fact, Java regards the following program as being identical to that of Figure 1.2.

`import turtles.*;public class BaseProgram extends TurtleProgram
{public void run(){}}`

Figure 1.2:
```
  1  import turtles.*;
  2  
  3  public class BaseProgram extends TurtleProgram {
  4      public void run() {
  5          // This is a simple baseline program.
  6      }
  7  }
```


Whitespaces (blank space) in your code are ignored by the compiler, which just skips over them while compiling the code.

**Review by rafaelgacesa — APPROVED** [review]
Sarah is mean 😢

**Review by Mistryr222 — APPROVED** [review]
Great!

**Pull request merged**